core/store/src/main/java/uk/gov/gchq/gaffer/store/operation/handler/named/AddNamedOperationHandler.java

```diff
@@ -40,20 +40,20 @@
 /**
  * Operation handler for AddNamedOperation which adds a Named Operation to the cache.
  */
-public class AddNamedOperationHandler implements OperationHandler<AddNamedOperation> {
+public class AddNamedOperationHandler implements OperationHandler<AddNamedOperation>, AddToCacheHandler<AddNamedOperation> {
```

Comment:
See other comments about Y being redundant when implementing X and Y, where Y is parent of interface X.